server/game/cards/locations/08/ashemark.js

```diff
@@ -0,0 +1,35 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class Ashemark extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onPhaseStarted: () => {
+                    this.gold = this.tokens['gold'];
```

Comment:
Something like that. It's very similar to #837, just with sacrifice instead of kill.